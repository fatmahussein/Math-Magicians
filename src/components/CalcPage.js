import Calculator from './calculator';

const CalcPage = () => (
  <div className="cal">
    <p className="maths">Lets do some Math!</p>
    <Calculator />
  </div>
);

export default CalcPage;
