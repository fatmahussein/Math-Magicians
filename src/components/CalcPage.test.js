import React from 'react';
import { render } from '@testing-library/react';
import CalcPage from './CalcPage';

describe('CalcPage Component', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<CalcPage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
