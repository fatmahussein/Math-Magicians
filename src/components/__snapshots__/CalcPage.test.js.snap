// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalcPage Component renders correctly 1`] = `
<DocumentFragment>
  <div
    class="cal"
  >
    <p
      class="maths"
    >
      Lets do some Math!
    </p>
    <div
      class="calc-wrapper"
    >
      <div
        class="input-wrapper"
        data-testid="initial"
      >
        0
      </div>
      <div
        class="row"
      >
        <button
          class="btn-wrapper"
          style="background-color: rgb(224, 224, 224);"
          type="button"
        >
          AC
        </button>
        <button
          class="btn-wrapper"
          style="background-color: rgb(224, 224, 224);"
          type="button"
        >
          +/-
        </button>
        <button
          class="btn-wrapper"
          style="background-color: rgb(224, 224, 224);"
          type="button"
        >
          %
        </button>
        <button
          class="btn-wrapper"
          style="background-color: rgb(245, 145, 62);"
          type="button"
        >
          รท
        </button>
      </div>
      <div
        class="row"
      >
        <button
          class="btn-wrapper"
          style="background-color: rgb(224, 224, 224);"
          type="button"
        >
          7
        </button>
        <button
          class="btn-wrapper"
          style="background-color: rgb(224, 224, 224);"
          type="button"
        >
          8
        </button>
        <button
          class="btn-wrapper"
          style="background-color: rgb(224, 224, 224);"
          type="button"
        >
          9
        </button>
        <button
          class="btn-wrapper"
          style="background-color: rgb(245, 145, 62);"
          type="button"
        >
          x
        </button>
      </div>
      <div
        class="row"
      >
        <button
          class="btn-wrapper"
          style="background-color: rgb(224, 224, 224);"
          type="button"
        >
          4
        </button>
        <button
          class="btn-wrapper"
          style="background-color: rgb(224, 224, 224);"
          type="button"
        >
          5
        </button>
        <button
          class="btn-wrapper"
          style="background-color: rgb(224, 224, 224);"
          type="button"
        >
          6
        </button>
        <button
          class="btn-wrapper"
          style="background-color: rgb(245, 145, 62);"
          type="button"
        >
          -
        </button>
      </div>
      <div
        class="row"
      >
        <button
          class="btn-wrapper"
          style="background-color: rgb(224, 224, 224);"
          type="button"
        >
          1
        </button>
        <button
          class="btn-wrapper"
          style="background-color: rgb(224, 224, 224);"
          type="button"
        >
          2
        </button>
        <button
          class="btn-wrapper"
          style="background-color: rgb(224, 224, 224);"
          type="button"
        >
          3
        </button>
        <button
          class="btn-wrapper"
          style="background-color: rgb(245, 145, 62);"
          type="button"
        >
          +
        </button>
      </div>
      <div
        class="row last"
      >
        <button
          class="btn-wrapper"
          style="background-color: rgb(224, 224, 224);"
          type="button"
        >
          0
        </button>
        <button
          class="btn-wrapper"
          style="background-color: rgb(224, 224, 224);"
          type="button"
        >
          .
        </button>
        <button
          class="btn-wrapper"
          style="background-color: rgb(245, 145, 62);"
          type="button"
        >
          =
        </button>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
